docs/reference/run.md

```diff
@@ -1385,30 +1385,71 @@ it will provide a named alias for the container being linked to.
 
 ### ENV (environment variables)
 
-When a new container is created, Docker will set the following environment
+On Linux, when a new container is created, Docker will set the following environment
 variables automatically:
 
 | Variable | Value |
 | -------- | ----- |
 | `HOME` | Set based on the value of `USER` |
 | `HOSTNAME` | The hostname associated with the container |
-| `PATH` | Includes popular directories, such as `:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin` |
+| `PATH` | Includes popular directories, such as `/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin` |
 | `TERM` | `xterm` if the container is allocated a pseudo-TTY |
 
+On Windows, when a new container is created, Docker will not set any environment
+variables automatically. 
+
+
 Additionally, the operator can **set any environment variable** in the
 container by using one or more `-e` flags, even overriding those mentioned
 above, or already defined by the developer with a Dockerfile `ENV`:
 
-    $ docker run -e "deep=purple" --rm ubuntu /bin/bash -c export
-    declare -x HOME="/"
-    declare -x HOSTNAME="85bc26a0e200"
-    declare -x OLDPWD
-    declare -x PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
-    declare -x PWD="/"
-    declare -x SHLVL="1"
-    declare -x deep="purple"
+```bash
+$ docker run -e "deep=purple" --rm ubuntu /bin/bash -c export
```

Comment:
Perhaps we should change this to a simpler example. The `-c export` output is a bit hard to read;

```bash
$ docker run -e "deep=purple" --rm alpine env
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
HOSTNAME=d2219b854598
deep=purple
HOME=/root
```